### PR TITLE
Fixed error about spectra num

### PR DESCRIPTION
**Pull request by daichengxin**

### CODE REVIEW

**Comment by jpfeuffer** [comment]
I am excited :) Do you have an example report after the update?

**Comment by ypriverol** [comment]
They, have but I think it can't be seen online if is not merge the PR?

File: pmultiqc/modules/proteomicslfq/proteomicslfq.py [hunk 1]
```diff
@@ -10,12 +10,21 @@
...
 import pandas as pd
+from collections import Counter
 import re
 from pyteomics import mztab, mzml, openms
 import os
+import sqlite3
 
 # Initialise the main MultiQC logger
 log = logging.getLogger(__name__)
+con = sqlite3.connect('./proteomicslfq.db')
```
Comment (ypriverol) [inline pmultiqc/modules/proteomicslfq/proteomicslfq.py:21]: @daichengxin for the examples, we have three files and only one proteomicslfq.db. I think the results of the three projects will be overlap inside the database?

**Review by ypriverol — CHANGES_REQUESTED** [review]

File: pmultiqc/modules/proteomicslfq/proteomicslfq.py (hunk 1 above)
Comment (daichengxin) [reply pmultiqc/modules/proteomicslfq/proteomicslfq.py:21]: Yes. A possible solution is to let users enter the name of this database themselves. What do you think? @ypriverol

Comment (daichengxin) [reply pmultiqc/modules/proteomicslfq/proteomicslfq.py:21]: But js code is not easy to operate

Comment (ypriverol) [reply pmultiqc/modules/proteomicslfq/proteomicslfq.py:21]: I guess the best way is to collect as the report.html or some kind of auto-generated name for the database?

Comment (daichengxin) [reply pmultiqc/modules/proteomicslfq/proteomicslfq.py:21]: js also needs to access the database, and the name of the database accessed after installing this package is fixed. js accesses the db file under the same file as the html report. I am trying another method, by using the -o parameter of multiqc to generate different folders to store different reports.

**Review by ypriverol — APPROVED** [review]

**Pull request merged**